### DISCUSSION TITLE
field required (type=value_error.missing)

### DISCUSSION THREAD

**Discussion by Ar-Anik** [fastapi/fastapi#6092] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import uvicorn
from fastapi import FastAPI
from pydantic import BaseModel, EmailStr

class rdUserBase(BaseModel):
    username: str
    email: EmailStr
    full_name: str = None

class rdUserIn(rdUserBase):
    password: str

class rdUserOut(rdUserBase):
    pass

class rdUserInDB(rdUserBase):
    hased_password: str

def rdFake_password_hasher(raw_password: str):
    return "supersecret" + raw_password

def rdFake_save_user(user_in: rdUserIn):
    hashed_password = rdFake_password_hasher(user_in.password)
    user_in_db = rdUserInDB(**user_in.dict(), hashed_password=hashed_password)
    print("User Saved! ..not really")
    return user_in_db

@app.post("/usr/", response_model=rdUserOut)
async def body2(user_in: rdUserIn):
    user_saved = rdFake_save_user(user_in)
    return user_saved

if __name__ == "__main__":
    uvicorn.run(app, host="0.0.0.0", port=int(os.getenv("9000")))
```

### Description

here give me value error but i try as like fast api documentation. when i first try multiple model it give me same type error.
error details : 
 in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for rdUserInDB
hased_password
  field required (type=value_error.missing)

### Wanted Solution

Yes

### Wanted Code

```python
Yes
```

### Operating System

Linux

### FastAPI Version

0.70.1

### Python Version

Python 3.8.10

**Comment by aybruhm** [comment, marked as answer]
Hello @Ar-Anik,

Kindly rename hased_password to hashed_password on your rdUserInDB model. 

Current: 

```python
class rdUserInDB(rdUserBase):
    hased_password: str
```

Updated: 

```python
class rdUserInDB(rdUserBase):
    hashed_password: str
```
